### PR TITLE
update envoy gateway status

### PR DESCRIPTION
**Pull request by arkodg**
Signed-off-by: Arko Dasgupta <arko@tetrate.io>

**What type of PR is this?**
/kind documentation

### CODE REVIEW

File: site-src/implementations.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ See [here][emissary-gateway-api] for more details on using the Gateway API with
 ### Envoy Gateway
 
 [Envoy Gateway][eg-home] is an subproject of the Envoy project for managing an Envoy-based Kubernetes-based application
```
Comment (danehans) [inline site-src/implementations.md:125]: @arkodg can you also fix the improper grammar in this PR?

s/an subproject/a subproject/

or reword:

"[Envoy Gateway][eg-home] is a project within the Envoy community for managing Envoy proxy as an application gateway."

**Comment by robscott** [comment]
Thanks @arkodg!
 
/ok-to-test
/approve

**Pull request merged**